src/omv/sensors/lepton.c

```diff
@@ -40,15 +40,21 @@
 #define VOSPI_FIRST_PACKET      (0)
 #define VOSPI_FIRST_SEGMENT     (1)
 #define LEPTON_TIMEOUT          (1000)
+// Temperatures in Celsius
 #define DEFAULT_MIN_TEMP        (-17.7778f)
```

Comment:
-17.77 is outside the sensors range, any chance I can make this -10.0 or am I missing something here?

Reply:
I'm not sure, CC @kwagyeman

Reply:
It's 0 in Fahrenheit. Please make it -10.

Reply:
What about the default max ? Could it need fixing too ? The measurement mode looks much better with the higher range, see my comment below.